docs/resources/rule_regex.md

```diff
@@ -63,10 +63,10 @@ resource "wallarm_rule_regex" "scanner_rule" {
 
 ## Argument Reference
 
-* `client_id` - (Optional) ID of the client to apply the rules to. The value is required for multi-tenant scenarios.
-* `attack_type` - (Required) Attack type that will be detected when the parameter value in the request matches the regular expression. Can be: `any`, `sqli`, `rce`, `crlf`, `nosqli`, `ptrav`, `xxe`, `ptrav`, `xss`, `scanner`, `redir`, `ldapi`.
-* `action` - (Optional) Rule conditions. Possible attributes are described below.
-* `point` - (Required) Request parts to apply the rules to. The full list of possible values is available in the [Wallarm official documentation](https://docs.wallarm.com/user-guides/rules/request-processing/#identifying-and-parsing-the-request-parts).
+* `client_id` - (optional) ID of the client to apply the rules to. The value is required for [multi-tenant scenarios][2].
+* `attack_type` - (**required**) attack type that will be detected when the parameter value in the request matches the regular expression. Can be: `any`, `sqli`, `rce`, `crlf`, `nosqli`, `ptrav`, `xxe`, `ptrav`, `xss`, `scanner`, `redir`, `ldapi`.
```

Comment:
there are more attack types in UI

Reply:
But only these are currently supported by the provider.